backend/src/main/java/com/cruru/question/service/QuestionService.java

```diff
@@ -42,23 +44,32 @@ private Question toQuestion(QuestionType type, QuestionCreateRequest request, Ap
         return new Question(type, request.question(), request.description(), request.orderIndex(), applyForm);
     }
 
-    public List<QuestionResponse> findByApplyFormId(Long applyFormId) {
-        return questionRepository.findAllByApplyFormId(applyFormId)
-                .stream()
-                .map(this::toQuestionResponse)
-                .toList();
+    public List<Question> findByApplyFormId(Long applyFormId) {
+        return questionRepository.findAllByApplyFormId(applyFormId);
     }
 
-    private QuestionResponse toQuestionResponse(Question question) {
-        List<ChoiceResponse> choiceResponses = choiceService.findAllByQuestionId(question.getId());
+    public List<QuestionResponse> toQuestionResponse(List<Question> questions) {
+        List<QuestionResponse> questionResponses = new ArrayList<>();
+        for (Question question : questions) {
+            List<ChoiceResponse> choiceResponses = getChoiceResponses(question);
+            questionResponses.add(new QuestionResponse(
+                    question.getId(),
+                    question.getQuestionType().name(),
+                    question.getContent(),
+                    question.getDescription(),
+                    question.getSequence(),
+                    choiceResponses
+            ));
+        }
+        return questionResponses;
+    }
```

Comment:
```suggestion
    public List<QuestionResponse> toQuestionResponses(List<Question> questions) {
        return questions.stream()
                .map(question -> getQuestionResponse(question))
                .toList();
    }

    private QuestionResponse getQuestionResponse(Question question) {
        return new QuestionResponse(
                question.getId(),
                question.getQuestionType().name(),
                question.getContent(),
                question.getDescription(),
                question.getSequence(),
                getChoiceResponses(question)
        );
    }
```

Stream을 통해 questionResponses 변경을 막을 수 있고(new ArrayList를 생성하기 때문에), 인덴트를 줄일 수 있을 것 같아요!